website/docs/docs/build/metrics-overview.md

```diff
@@ -20,7 +20,8 @@ The keys for metrics definitions are:
 | `config` | Provide the specific configurations for your metric.   | Optional |
 | `label` | The display name for your metric. This value will be shown in downstream tools.   | Required |
 | `filter` | You can optionally add a filter string to any metric type, applying filters to dimensions, entities, or time dimensions during metric computation. Consider it as your WHERE clause.   | Optional |
-
+| `fill_nulls_with` | Set the value in your metric definition instead of null (such as zero).| Optional |
+| `join_to_timespine` | Boolean that indicates if the aggregated measure should be joined to the time spine table to fill in missing dates. Default `false`. | Optional |
```

Comment:
added to the parameters table

Reply:
@mirnawong1 I think we should remove these. They aren't included in the metric schema. They are additional parameters you can add to the measure input.